<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>639</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PHP蚊子</string>
  </property>
  <widget class="QWidget" name="main">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QWidget" name="wen_jian_guan_li_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,50">
         <item>
          <widget class="QLabel" name="shelllabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <column>
              <property name="text">
               <string>我的电脑</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidget_2">
             <column>
              <property name="text">
               <string>文件名</string>
              </property>
              <property name="textAlignment">
               <set>AlignLeading|AlignVCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>修改时间</string>
              </property>
              <property name="textAlignment">
               <set>AlignLeading|AlignVCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>权限</string>
              </property>
              <property name="textAlignment">
               <set>AlignLeading|AlignVCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>大小</string>
              </property>
              <property name="textAlignment">
               <set>AlignLeading|AlignVCenter</set>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="bei_jing_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="cmd_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="cmd_textEdit">
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1,1">
           <item>
            <widget class="QLineEdit" name="cmd_lineEdit">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="zhi_xing_pushButton">
             <property name="text">
              <string>执行</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear_pushButton">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="shell_guan_li_action"/>
    <addaction name="wen_jian_guan_li_action"/>
    <addaction name="cmd_action"/>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>快捷功能</string>
    </property>
    <addaction name="cha_xun_users_action"/>
    <addaction name="tian_jia_users_actiond"/>
    <addaction name="actiondsf"/>
    <addaction name="shutdown_action"/>
    <addaction name="actionge"/>
    <addaction name="open_3389_action"/>
    <addaction name="love_me_action"/>
    <addaction name="actionaa_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="wen_jian_guan_li_action">
   <property name="text">
    <string>文件管理</string>
   </property>
  </action>
  <action name="cmd_action">
   <property name="text">
    <string>虚拟终端</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>setting</string>
   </property>
  </action>
  <action name="actionadsf">
   <property name="text">
    <string>adsf</string>
   </property>
  </action>
  <action name="shell_guan_li_action">
   <property name="text">
    <string>shell管理</string>
   </property>
  </action>
  <action name="cha_xun_users_action">
   <property name="text">
    <string>查询用户</string>
   </property>
  </action>
  <action name="tian_jia_users_actiond">
   <property name="text">
    <string>添加用户</string>
   </property>
  </action>
  <action name="shutdown_action">
   <property name="text">
    <string>强制关机</string>
   </property>
  </action>
  <action name="actiondsf">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionge">
   <property name="text">
    <string>格式化C盘</string>
   </property>
  </action>
  <action name="open_3389_action">
   <property name="text">
    <string>开启3389</string>
   </property>
  </action>
  <action name="love_me_action">
   <property name="text">
    <string>联系作者</string>
   </property>
  </action>
  <action name="actionaa_2">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
